packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -343,6 +363,12 @@ export interface IContainerRuntimeBaseExperimental extends IContainerRuntimeBase
  * @alpha
  */
 export interface IFluidDataStoreChannel extends IDisposable {
+	/**
+	 * Optional policies that can be applied to the data store channel.
```

Comment:
I like to take every opportunity to clarify the direction of control for these interfaces/props. So would it be correct to say

```suggestion
	 * Optional policies that the data store channel may adhere to that the data store context should know about.
```